samples/bluetooth/throughput/prj_nrf5340dk_nrf5340_cpuapp.conf

```diff
@@ -1,39 +0,0 @@
-#
```

Comment:
This can't be helped because the controller is on the other core, it is handled by the child image part. The warning appears even in upstream zephyr samples when compiling for nrf5340

Reply:
That is why we have here prj_board.conf files which replace the prj.conf file for nrf5340 to avoid cmake warnings.
I don't see a point to replace working configuration with one which generates build system warnings

Reply:
There were 3 reported issue in the space of a week with `prj_<board>.conf` files, 2 from ncs and 1 from zephyr who all had different issues but were all caused by using the old deprecated file. One had overlay files that were not being included, one was having a false build type set and I can't remember what the upstream issue was - they were all caused by using this outdated file and these files should never have been accepted into ncs, this way of configuring applications was how zephyr 1.x was configured.